tests/Data/ProgrammesDb/EntityRepository/PromotionRepository/FindActivePromotionsByPidTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Tests\BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\CoreEntityRepository;
+
+use BBC\ProgrammesPagesService\Domain\ValueObject\Pid;
+use DateTimeImmutable;
+use Tests\BBC\ProgrammesPagesService\AbstractDatabaseTest;
+
+/**
+ * @covers BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\PromotionRepository::<public>
+ */
+class FindActivePromotionsByPidTest extends AbstractDatabaseTest
+{
+    /**
+     * @var PromotionRepository
+     */
+    private $promotionRepository;
+
+    public function setUp()
```

Comment:
If you've only got a single test function is there much point having a setUp? The function body could be placed directly into the test function